src/stack_limiter/thunk.rs

```diff
@@ -1,14 +1,14 @@
-#[cfg(not(features = "std"))]
+#[cfg(not(feature = "std"))]
 use alloc::collections::BTreeMap as Map;
 use alloc::vec::Vec;
 use parity_wasm::{
 	builder,
-	elements::{self, FunctionType, Internal},
+	elements::{self, FunctionType, Instruction, Internal},
 };
-#[cfg(features = "std")]
-use std::collections::HashMap as Map;
+#[cfg(feature = "std")]
+use std::collections::BTreeMap as Map;
```

Comment:
why changed to btree_map?

Reply:
Our CI has "check  fuzzer reproduction" step. This step will create 2 wasm binaries and compare `ext1` to `ext2`. So, this test fails due to `RandomState` in `HashMap<K, V, S = RandomState>` (rust hash maps randomize keys to prevent some memory attacks iirc).

gwasm-instrument::inject will produce different outputs from one input.